specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2020-03-01/managedClusters.json

```diff
@@ -1562,6 +1566,15 @@
       },
       "description": "AgentPoolType represents types of an agent pool."
     },
+    "AgentPoolUpgradeSettings": {
```

Comment:
what would happen if this property is set on the first call and then missing from the request body on subsequent call?

Reply:
Then we should leave it as is. You can't null it out once set. Is there a best practice here?